src/BibleWell.Platform.Maui/NotificationRegistrationService.cs

```diff
@@ -0,0 +1,80 @@
+using System.Text.Json;
+using BibleWell.Aquifer;
+using BibleWell.PushNotifications;
+using Microsoft.Maui.Storage;
+
+namespace BibleWell.Platform.Maui;
```

Comment:
This class seems like it should belong in the `BibleWell` project.  The only MAUI part of this is the use of `SecureStorage`.  For that, I'd make something similar to the `IUserPreferencesService` and its implementations but named something like `ISecureUserStorageService`.  Then you can implement _that_ here in the `BibleWell.Platform.Maui` project.